packages/integration-tests/src/test-in-memory/data_storage_auth_with_triggers.test.ts

```diff
@@ -60,34 +58,3 @@ void it('data storage auth with triggers', () => {
   ]);
   /* eslint-enable spellcheck/spell-checker */
 });
-
-void it('data without auth with lambda auth mode', () => {
```

Comment:
does putting each of these tests in their own `define` block resolve the call count issue? If not, this solution is fine

Reply:
tried that locally and its still not happy.